### DISCUSSION TITLE
Upcoming beta version

### DISCUSSION THREAD

**Discussion by albaintor** [albaintor/homeassistant_electrolux_status#5] [General]
Hi,
I spent some time on the new integration and now I can get the entities. This is a start but it is getting better...
<img width="526" alt="image" src="https://github.com/albaintor/homeassistant_electrolux_status/assets/118518828/da6b1778-12e5-4570-b714-729f0c8af4c6">

**Comment by slingshot74** [comment]
@albaintor nice is there a possibility I can test it for you?

**Comment by albaintor** [comment]
Sure I’ll publish a release tomorrow
